worker/src/main/java/gov/cms/ab2d/worker/service/FileServiceImpl.java

```diff
@@ -1,8 +1,11 @@
 package gov.cms.ab2d.worker.service;
 
 import lombok.extern.slf4j.Slf4j;
+import org.bouncycastle.jcajce.provider.digest.SHA256;
```

Comment:
If we remove bouncy castle, `org.apache.commons.codec.digest.DigestUtils` can be used instead.